main.js

```diff
@@ -1,26 +1,32 @@
 var phrase = 'grumpy wizards make toxic brew for the evil queen'
 var array = []
 var $phraseContainer = document.getElementsByTagName('div')[1]
+
 for(var c = 0; c < phrase.length; c++) {
   var obj = {
     letter: ''
   }
   obj.letter = phrase[c]
   array.push(obj)
 }
+array.push({
+  currentIndex: 0
+})
 
-function renderChar(obj) {
+function renderChar(obj, cVal) {
   var $currentChar = document.createElement('span')
   $currentChar.classList.add('char')
   $currentChar.textContent = obj.letter
+  if( array[array.length - 1].currentIndex === cVal )
```

Comment:
Try to write this function so it only uses its parameters and the `document` object to get its work done.